aws.go

```diff
@@ -31,7 +32,7 @@ func queryUserData() (*UserData, error) {
 	return userData, err
 }
 
-func queryMetaData(url, name string) (string, error) {
+func queryMetaData(url string) (string, error) {
 	// http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html#instancedata-data-retrieval
 	// call http://169.254.169.254/latest/meta-data/instance-id with httpbackoff
```

Comment:
True, that would be surprising! But even if only one in a million times it would fail, I think it would make sense to retry (and the code difference is just `http.Get(url)` vs `httpbackoff.Get(url)`.